packages/server-side-render/src/hook.ts

```diff
@@ -35,7 +43,7 @@ export function removeBlockSupportAttributes( attributes ) {
 		spacing,
 		typography,
 		...restStyles
-	} = attributes?.style || {};
+	} = ( attributes?.style as Record< string, unknown > ) || {};
```

Comment:
We can then revert this change
```suggestion
	} = attributes?.style || {};
```